app/styles/_state.scss

```diff
@@ -36,6 +36,11 @@
   }
 }
 
+#suggest-sync{
+  padding-top: 14px;
+  padding-bottom: 14px;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/128755/16456257/5046cebc-3de5-11e6-99f3-eaa1c0fb05c5.png)

remove this new style, add `pad` to `info nudge` in the new panel.